algoliasearch-common/src/main/java/com/algolia/search/objects/TimeRange.java

```diff
@@ -0,0 +1,34 @@
+package com.algolia.search.objects;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import java.io.Serializable;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class TimeRange implements Serializable {
+
+  private long from;
+  private long until;
```

Comment:
I know it's a bit longer to implement, but from a user PoV, I guess it would be easier if we were to provide [Java 8 `Time` instances](https://www.baeldung.com/java-8-date-time-intro) rather than `long`s. I've already written `CustomZonedDateTimeSerializer`/`CustomZonedDateTimeDeserializer` for the `ABTest.java` that you can probably reuse.

Reply:
Ok! nice I'll update it.